eclass/elisp-common.eclass

```diff
@@ -380,7 +362,7 @@ elisp-modules-install() {
 	shift
 	# Don't bother inheriting multilib.eclass for get_libdir(), but
 	# error out in old EAPIs that don't support it natively.
-	[[ ${EAPI} == [45] ]] \
+	[[ ${EAPI} == 5 ]] \
 		&& die "${ECLASS}: Dynamic modules not supported in EAPI ${EAPI}"
 	ebegin "Installing dynamic modules for GNU Emacs support"
 	( # subshell to avoid pollution of calling environment
```

Comment:
(more a comment): isn't this a bit brittle, seeing that only EAPI>=7 handles errors in subshells correctly?

Reply:
Hm, unless I'm missing something, this test happens before the subshell is spawned.